From adba2ebc116bc5baf5bb0793e323e173340d11c9 Mon Sep 17 00:00:00 2001
From: Matthias Clasen <mclasen@redhat.com>
Date: Sun, 31 Mar 2013 20:28:19 -0400
Subject: [PATCH] info: Switch around GNOME and distro information

This makes the distribution logo prominent, and puts GNOME version
information in the small print.

https://bugzilla.gnome.org/show_bug.cgi?id=695691

http://pkgs.fedoraproject.org/rpms/control-center/blob/master/f/distro-logo.patch

Edited by Jeremy Bicha to use a larger Ubuntu font for the Ubuntu version
and drop the "Version" string from the GNOME version line
---
 panels/info/cc-info-overview-panel.c |    9 +++------
 panels/info/info-overview.ui         |   16 ++++++++++------
 2 files changed, 13 insertions(+), 12 deletions(-)

--- a/panels/info/cc-info-overview-panel.c
+++ b/panels/info/cc-info-overview-panel.c
@@ -797,7 +794,7 @@ info_overview_panel_setup_overview (CcIn
   if (res)
     {
       g_autofree gchar *text = NULL;
-      text = g_strdup_printf (_("Version %s"), priv->gnome_version);
+      text = g_strdup_printf ("%s", priv->gnome_version);
       gtk_label_set_text (GTK_LABEL (priv->version_label), text);
     }
 
--- a/panels/info/info-overview.ui
+++ b/panels/info/info-overview.ui
@@ -12,13 +12,14 @@
       <object class="GtkBox">
         <property name="visible">True</property>
         <property name="can_focus">False</property>
-        <property name="spacing">18</property>
+        <property name="spacing">6</property>
         <property name="orientation">vertical</property>
         <child>
           <object class="GtkImage" id="system_image">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
-            <property name="resource">/org/gnome/control-center/info/GnomeLogoVerticalMedium.svg</property>
+            <property name="pixel_size">128</property>
+            <property name="icon_name">ubuntu-logo-icon</property>
           </object>
           <packing>
             <property name="expand">False</property>
@@ -27,13 +28,16 @@
           </packing>
         </child>
         <child>
-          <object class="GtkLabel" id="version_label">
+          <object class="GtkLabel" id="os_name_label">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
             <property name="label">Version 3.0</property>
             <property name="selectable">True</property>
+            <property name="margin-bottom">24</property>
             <attributes>
               <attribute name="scale" value="1.25"/>
+              <attribute name="weight" value="bold"/>
+              <attribute name="font-desc" value="Ubuntu 16"/>
             </attributes>
           </object>
           <packing>
@@ -118,8 +122,8 @@
                 <property name="visible">True</property>
                 <property name="can_focus">False</property>
                 <property name="xalign">1</property>
-                <property name="label" translatable="yes" comments="To translators: this field contains the distro name and version">OS name</property>
-                <property name="mnemonic_widget">os_name_label</property>
+                <property name="label">GNOME</property>
+                <property name="mnemonic_widget">version_label</property>
                 <style>
                  <class name="dim-label"/>
                 </style>
@@ -228,7 +232,7 @@
               </packing>
             </child>
             <child>
-              <object class="GtkLabel" id="os_name_label">
+              <object class="GtkLabel" id="version_label">
                 <property name="visible">True</property>
                 <property name="can_focus">False</property>
                 <property name="xalign">0</property>
